kiva/fonttools/font_manager.py

```diff
@@ -107,7 +107,15 @@ def __init__(self):
                 break
         else:
             # use anything
-            self.default_font["ttf"] = ttffiles[0]
+            if len(ttffiles) > 0:
+                self.default_font["ttf"] = ttffiles[0]
+            else:  # no available fonts, use TestTTF font
+                import pkg_resources
+                data_dir = pkg_resources.resource_filename(
+                    "kiva.fonttools.tests", "data"
+                )
+                path = os.path.join(data_dir, "TestTTF.ttf")
```

Comment:
I don't know very much about fonts, and I imagine this `TestTTF.ttf` is nott going to be useful as a default.  At a minimum though, this prevents the test from failing.

If this font isn't suitable for downstream needs, then one could use `add_application_font` (which previously would fail because it required initializing a font manager.

Reply:
I'm not sure this is a good solution. I think it might be better just to emit a warning and leave the default font unset. This `TestTTF.ttf` font isn't much more than a file header. It's extremely minimal.